pkg/tinydtls/contrib/sock_dtls.c

```diff
@@ -345,6 +345,24 @@ void sock_dtls_session_destroy(sock_dtls_t *sock, sock_dtls_session_t *remote)
     dtls_close(sock->dtls_ctx, &remote->dtls_session);
 }
 
+void sock_dtls_session_get_udp_ep(const sock_dtls_session_t *session,
+                                  sock_udp_ep_t *ep)
+{
+    assert(session);
+    assert(ep);
+
+    _session_to_ep(&session->dtls_session, ep);
```

Comment:
Would it make sense to replace the calls of this function with `sock_dtls_session_get_udp_ep()` in general?